### PR TITLE
NZBGet 15.0

### PR DESCRIPTION
**Pull request by skithund**

### CODE REVIEW

File: Library/Formula/nzbget.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Nzbget < Formula
   desc "Binary newsgrabber for nzb files"
   homepage "http://nzbget.net/"
-  url "https://downloads.sourceforge.net/project/nzbget/nzbget-stable/14.2/nzbget-14.2.tar.gz"
-  sha1 "25adf5565d228cf1cbb8fa305732f61a6f869aa0"
+  url "https://downloads.sourceforge.net/project/nzbget/nzbget-stable/15.0/nzbget-15.0.tar.gz"
+  sha1 "f86447246faf7f9dd745be789caba2a9cfa7870a"
```
Comment (dunn) [inline Library/Formula/nzbget.rb]: Can you bump this up to `sha256`, please?

Comment (skithund) [inline Library/Formula/nzbget.rb]: Done

File: Library/Formula/nzbget.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Nzbget < Formula
   desc "Binary newsgrabber for nzb files"
   homepage "http://nzbget.net/"
-  url "https://downloads.sourceforge.net/project/nzbget/nzbget-stable/14.2/nzbget-14.2.tar.gz"
-  sha1 "25adf5565d228cf1cbb8fa305732f61a6f869aa0"
+  url "https://downloads.sourceforge.net/project/nzbget/nzbget-stable/15.0/nzbget-15.0.tar.gz"
+  sha256 "3ef13f3e5917e4cda19c4fc0cd37e79967a19b4e3448c239ff24e37712a6cc0a"
 
   devel do
```
Comment (DomT4) [inline Library/Formula/nzbget.rb]: Is `devel` is now older than `stable`? Remove it if so.

Comment (skithund) [inline Library/Formula/nzbget.rb]: Removed.
Also updated `head`

**Comment by DomT4** [comment]
The `libpar2` resource needs updating to `sha256` as well. Otherwise, looks great.

**Comment by skithund** [comment]
Done that too.

**Comment by dunn** [comment]
Merged in e276ff4f0120445e64125a87509dbd6ff3f8c561, thank you!

**Pull request closed without merging**